src/Compliance/AuditReports/AuditReports.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Demonstrates how to leverage AuditReports feature.</Description>
+    <OutputType>Exe</OutputType>
+
+    <!-- This property is used to control whether compliance report generator kicks in -->
+    <!--<GenerateComplianceReport>true</GenerateComplianceReport>-->
+
+    <!-- 
+        Optionally you can control where the "ComplianceReport.json" file should be created.
+        If this value is changed, make sure to make a corresponding change in Program.cs.
+     -->
+    <!--<ComplianceReportOutputPath>$(MSBuildProjectDirectory)\$(TargetFramework)\</ComplianceReportOutputPath>-->
```

Comment:
❔ Just occurred to me, would this work on linux?

Reply:
Yes, just tested it